### PR TITLE
[WIP] Fix meters in P3B5

### PR DESCRIPTION
**Pull request by yngtodd**
There were a couple bits in the `P3B5` benchmark that were not updated when `DARTS` was moved to common. This fixes that issue.

### CODE REVIEW

**Comment by jmohdyusof** [comment]
It looks like it is generating this error, I think you fixed this in the darts examples? 
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule.See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate

**Comment by jmohdyusof** [comment]
Also can you see my comment on the darts examples, still an issue that needs fixing.
